<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSettingWidget</class>
 <widget class="QWidget" name="qSettingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="qTitleLabel">
     <property name="styleSheet">
      <string notr="true">font: 500 20pt &quot;思源黑体 CN Medium&quot;;</string>
     </property>
     <property name="text">
      <string>Setting</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="qFormLayout">
     <property name="horizontalSpacing">
      <number>30</number>
     </property>
     <property name="leftMargin">
      <number>20</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="qWeightSelectLabel">
       <property name="minimumSize">
        <size>
         <width>101</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 500 10pt &quot;思源黑体 CN Medium&quot;;</string>
       </property>
       <property name="text">
        <string>Weight Select</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="qDynamicWeightLabel">
       <property name="minimumSize">
        <size>
         <width>101</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 500 10pt &quot;思源黑体 CN Medium&quot;;</string>
       </property>
       <property name="text">
        <string>Dynamic Weight</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="SwitchButton" name="qDynamicWeightSwitchButton" native="true">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="SwitchButton" name="qWeightSelectSwitchButton" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="qLanguageLabel">
       <property name="minimumSize">
        <size>
         <width>101</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 500 10pt &quot;思源黑体 CN Medium&quot;;</string>
       </property>
       <property name="text">
        <string>Language</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="qChineseRadioButton">
         <property name="minimumSize">
          <size>
           <width>68</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Chinese</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="qEnglishRadioButton">
         <property name="minimumSize">
          <size>
           <width>68</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>English</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="qOKPushButton">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SwitchButton</class>
   <extends>QWidget</extends>
   <header location="global">inc/switchButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
